kolibri/plugins/perseus_viewer/perseusBuildConfig.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * This file defines additional webpack configuration for this plugin.
+ * It will be bundled into the webpack configuration at build time.
+ */
+var path = require('path');
+var webpack = require('webpack');
+
+const submodules = path.resolve(__dirname, '..', 'submodules')
+
+const perseus_node_modules = path.resolve(submodules, 'perseus', 'node_modules');
+
+const node_modules = path.resolve(__dirname, './node_modules');
+
+const base_node_modules = path.resolve(__dirname, '../../../node_modules');
+
+module.exports = {
+  entry: path.resolve(__dirname, './submodules/perseus/src/perseus.js'),
+  output: {
+    path: path.resolve(__dirname, './assets/dist'),
+    filename: 'perseus.js',
```

Comment:
This file contains references to the developer machine, which will lead to undesirable diffs when run on a new machine:

```diff
- // EXTERNAL MODULE: /home/richard/github/kolibri/node_modules/inline-style-prefixer/static/plugins/filter.js
+ // EXTERNAL MODULE: /Users/d/Projects/le/kolibri/node_modules/inline-style-prefixer/static/plugins/filter.js
```


aside: kind of cool that our `./kolibri` directories are exactly the same number of characters into their respective absolute paths...

Reply:
Oh weird - that's not helpful at all.